lib/convection/model/template/resource/aws_lambda.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../resource'
```

Comment:
Can you rename this file to `aws_lambda_function.rb` to fit the current convention?